src/gui/qgsexternalresourcewidget.h

```diff
@@ -147,12 +147,17 @@ class GUI_EXPORT QgsExternalResourceWidget : public QWidget
     //! emitteed as soon as the current document changes
     void valueChanged( const QString & );
 
+  protected:
+    //! Needed to block events when showing a dialog
```

Comment:
```suggestion
    // Needed to block events when showing a dialog
```

(we usually skip over doxygen for overridden methods)